packages/data-stores/src/queries/use-launchpad-navigator.ts

```diff
@@ -0,0 +1,44 @@
+import { useQuery } from '@tanstack/react-query';
```

Comment:
👋🏻 Just a quick note that over at @Automattic/team-calypso-frameworks we think that the `data-stores` package is not suitable for custom RQ query hooks (they're not data stores after all). We think that queries would be best stored as close to the UI (where it's used) as possible. We know there are still a few queries located in the `data-stores` package but we'd like to have them moved to a more fitting location.